app/controllers/api/v1/bookmarks_controller.rb

```diff
@@ -0,0 +1,117 @@
+class Api::V1::BookmarksController < Api::V1::BaseController
```

Comment:
Shouldn't this be ExternalBookmarksController? We may well want to add API for regular bookmarks at some point